src/ucs/sys/compiler_def.h

```diff
@@ -87,6 +87,10 @@
 /* Used for labels */
 #define UCS_EMPTY_STATEMENT {}
 
+/* Helper macro for address arithmetic in bytes */
+#define UCS_PTR_BYTE_OFFSET(_ptr, _offset) \
```

Comment:
we already have such a macro in `compiler.h`

Reply:
i moved it to compiler_def.h so strided_alloc.h could be installed

Reply:
right, missed that you removed that